docs/examples/201/aci-sftp-files/main.bicep

```diff
@@ -0,0 +1,130 @@
+param storageAccountType string = 'Standard_LRS'
+param fileShareName string = 'sftpfileshare'
+param sftpUser string
+param sftpPassword string {
+  secure: true
+}
+param location string = resourceGroup().location
+
+var scriptName = 'createFileShare'
+var identityName = 'scratch'
+var roleDefinitionId = resourceId('Microsoft.Authorization/roleDefinitions', 'b24988ac-6180-42a0-ab88-20f7382dd24c')
+var roleDefinitionName = guid(identityName, roleDefinitionId)
```

Comment:
this is the roleAssignmentName, not the roleDefinitionName, yeah?

Reply:
Correct!